packages/webapp/package.json

```
@@ -26,7 +26,7 @@
     "graphql": "^15.8.0",
     "graphql-request": "^3.7.0",
     "idb-keyval": "^5.1.5",
-    "next": "14.2.9",
+    "next": "15.0.1-canary.1",
```

```suggestion
    "next": "15.0.0",
```
Don't think you meant to use canary version?

---

though I think v15.0.1 is coming later today or tomorrow

---

Ah it's just what the codemod sets 😂